policy-ansible-for-nsxt/module_utils/policy_edge_node.py

```diff
@@ -22,4 +22,4 @@ class PolicyEdgeNode(object):
     @staticmethod
     def get_resource_base_url(site_id, enforcementpoint_id, edge_cluster_id):
         return '/infra/sites/{}/enforcement-points/{}/edge-clusters/{}/edge-nodes'.format(
-+            site_id, enforcementpoint_id,edge_cluster_id)
+            site_id, enforcementpoint_id,edge_cluster_id)
```

Comment:
maybe change it to:
site_id, enforcementpoint_id, edge_cluster_id
Space before edge_cluster_id